Form-Controls/styles.css

```diff
@@ -0,0 +1,232 @@
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300..800;1,300..800&display=swap');
+
+* {
+    box-sizing: border-box;
+}
+
+
+body {
+    background-color: #dbdadd;
+    font-family: 'open sans', sans-serif;
+    display: block;
+    align-items: center;
+    justify-content: center;
+    min-height: 100vh;
+    margin: 0;
+    text-align: center;
+
+}
+
+/* ------------------- header ----------------------- */
+
+header {
+    margin: 1rem 4rem;
+}
+
+/* ------------------- main-container ----------------------- */
+
+main {
+    margin-top: 1.5rem;
+    margin-bottom: 5rem;
+}
+
+.container {
+    background-color: #fff;
+    border-radius: 5px;
+    box-shadow: 0 2px 10px;
+    width: 580px;
+    margin: auto;
+}
+
+.form {
+    padding: 30px 40px;
+}
+
+.form-control {
+    margin-bottom: 10px;
+    padding-bottom: 20px;
+    position: relative;
+}
+
+.form-control label {
+    color: #777;
+    display: block;
+    margin-bottom: 5px;
+
+}
+
+.form-control input {
+    border: 2px solid #f0f0f0;
+    border-radius: 4px;
+    display: block;
+    width: 100%;
+    padding: 10px;
+    font-size: 14px;
+}
+
+.form-control input:focus{
+    outline: 0;
+    border-color: #777;
+}
+
+/* ------------------- validation email ----------------------- */
+
+input:invalid:focus {
+    border-color:red;
+    }
+    
+input:valid:focus {
+    border-color: green;
+    }
+
+
+/* ------------------- radio button size ----------------------- */
+
+.form-size,
+.dropdown {
+    border: 2px solid #f0f0f0;
+    border-radius: 9px;
+    display: block;
+    width: 100%;
+    padding: 15px;
+    font-size: 14px;
+}
+
+#red {
+    accent-color: red;
+}
+
+#yellow {
+    accent-color: yellow;
+}
+
+#Blue {
+    accent-color: blue;
+}
+
+/* ------------------- Select dropdown ----------------------- */
+
+select {
+    padding: .5rem;
+    font-size: 12px;
+    color: #ffffff;
+    background-color: #3498db;
+    border: none;
+    cursor: pointer;
+    width: 15%;
+}
+
+option {
+    background-color: #f9f9f9;
+    color: black;
+    padding: 10px 10px;
+}
+
+.select:focus {
+    outline: none;
+    background-color: #3e8e41;
+}
+
+select:hover {
+    background-color: rgba(8, 249, 217, 0.357);
+    color: black;
+    font-weight: bold !important;
+}
+
+/* ------------------- form-control calendar ----------------------- */
+
+#date{
```

Comment:
Hi @Grajales-K ,
Should you choose to change the "id" to a "class" attribute, then you may have to adjust your css to avoid styling issues.